src/components/v1/V1Create/index.tsx

```diff
@@ -578,15 +582,25 @@ export default function V1Create() {
             setProjectFormModalVisible(false)
           }}
         >
-          <ProjectForm
-            form={projectForm}
-            onSave={async () => {
-              await projectForm.validateFields()
-              viewedCurrentStep()
-              onProjectFormSaved()
-              setProjectFormModalVisible(false)
-            }}
-          />
+          <Space direction="vertical" size="large">
+            <h1>
+              <Trans>Project details</Trans>
+            </h1>
+            <p>
+              <Trans>
+                Changes to these attributes can be made at any time and will be
+                applied to your project immediately.
+              </Trans>
+            </p>
+            <ProjectDetailsForm
+              form={projectForm}
+              onFinish={() => {
```

Comment:
Note that `await projectForm.validateFields()` has been removed. Antd already calls this function before its `onFinish` callback is called, so calling `validateFields` again here would be redundant.

Reply:
beautiful! being able to remove click handlers and externally handled validation feels like a big win. great call